linkerd/app/outbound/src/http/logical.rs

```diff
@@ -47,55 +47,60 @@ where
     let watchdog = cache_max_idle_age * 2;
 
     svc::stack(endpoint.clone())
-        .check_new_service::<Endpoint, http::Request<http::BoxBody>>()
         .push_on_response(
             svc::layers()
-                .push(svc::layer::mk(svc::SpawnReady::new))
+                .push(http::BoxRequest::layer())
```

Comment:
just out of curiosity, why put the box above metrics now?

Reply:
mostly I want the box as close to the inner service as possible, as it's responsible for satisfying the type signature of the inner service.